pkg/spec/createconfig.go

```diff
@@ -309,13 +309,17 @@ func (c *CreateConfig) getContainerCreateOptions(runtime *libpod.Runtime, pod *l
 	options = append(options, libpod.WithStopSignal(c.StopSignal))
 	options = append(options, libpod.WithStopTimeout(c.StopTimeout))
 
-	logPath, logTag := getLoggingOpts(c.LogDriverOpt)
+	logSizeMax, logPath, logTag := getLoggingOpts(c.LogDriverOpt)
+	intlogsizemax, _ := strconv.ParseInt(logSizeMax, 10, 64)
```

Comment:
need to handler the err😁

Reply:
Thanks for your review comments. Added error handling for string to int64 conversion in my this branch. please review it. i am not able to run currently make validation and make link in my local repo for testing.  Below are the error thrown by Make utility

~/go/src/github.com/containers/podman:supportforlogsizemax$ make validate
find . -name '*.go' -type f \
	-not \( \
		-name '.golangci.yml' -o \
		-name 'Makefile' -o \
		-path './vendor/*' -prune -o \
		-path './contrib/*' -prune \
	\) -exec gofmt -d -e -s -w {} \+
open _output/pkg/mod/github.com/varlink/go@v0.0.0-20190502142041-0f1d566d194b/cmd/varlink-go-certification/main.go.415909566: permission denied
open _output/pkg/mod/github.com/varlink/go@v0.0.0-20190502142041-0f1d566d194b/varlink/bridge_windows.go.285677317: permission denied
make: *** [Makefile:173: gofmt] Error 1


make lint
VERSION=1.18.0 GOBIN=/home/pmoogi/go/src/github.com/containers/podman/_output/bin sh ./hack/install_golangci.sh
Using existing /home/pmoogi/go/src/github.com/containers/podman/_output/bin/golangci-lint has version 1.18.0 built from 31afdf8 on 2019-09-10T09:35:37Z
hack/golangci-lint.sh run

Running golangci-lint for tunnel
Build Tags tunnel: apparmor,seccomp,selinux,remote,varlink,remoteclient
Skipped directories tunnel: pkg/api
hack/golangci-lint.sh: line 24: golangci-lint: command not found
make: *** [Makefile:169: golangci-lint] Error 127

--------------------------
Can you please help to resolve above issues? Post this i will submit PR.

Reply:
\resolved